### ISSUE TITLE
NVDA not announcing modal headings 

### ISSUE THREAD

**Issue by Githubuser4329** [nvaccess/nvda#12906]
### Steps to reproduce:
1) Open a modal with a heading and body text 
2) On open of the modal, focus goes to H1 inside the modal, but the heading is not read out 

### Actual behavior:
Modal header not being read out 

### Expected behavior:
Header should be read out by the screen reader 

#### NVDA installed/portable/running from source:
Installed 

#### NVDA version:
2021.2 and 2021.1 
#### Windows version:
Windows 10 

#### Name and version of other software in use when reproducing the issue:
Chrome (latest version) 

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, 2020.4 and 2021.1 and 2021.2

**Comment by feerrenrut** [comment]
Please provide a test sample / HTML snippet.

**Comment by ehollig** [comment]
Closing as stale
